sample/registry/s3-credentials.yaml

```
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: s3-credentials
+  namespace: llmos-system
+type: Opaque
+data:
+  # echo -n 89Z7T3kO21qiiFFwM7j6 | base64
+  # use echo -n to avoid the trailing newline
+  accessKeyID: 89Z7T3kO21qiiFFwM7j6
+  accessKeySecret: VmFsYU9JclEwRXVWODB1bHdud0YxbkZNbE9JSGpYZjRsQ0ZMQWhpRw==
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)